jwql/website/apps/jwql/views.py

```diff
@@ -194,6 +194,43 @@ def archive_thumbnails_ajax(request, inst, proposal):
     data = thumbnails_ajax(inst, proposal)
 
     return JsonResponse(data, json_dumps_params={'indent': 2})
+ 
+
+def authorize(request):
```

Comment:
I might suggest putting this new function, along with `login` and `logout` into a separate module, rather than within `views.py`. I think that might make more organizational sense.

Reply:
@laurenmarietta Good idea, that makes sense.  I probably can just stick them in `oauth.py` so all of the authentication stuff is contained in one place.